flare/agent_zoo/simple_rl_agents.py

```diff
@@ -11,23 +11,30 @@ class SimpleRLAgent(Agent):
     policy or off-policy RL algorithms.
     """
 
-    def __init__(self, env, num_games):
+    def __init__(self, env, num_games, action_noise=None):
         super(SimpleRLAgent, self).__init__(env, num_games)
+        self.action_noise = action_noise
 
     def _cts_store_data(self, observations, actions, states, rewards):
         assert len(observations) == 1 and len(actions) == 1
         self._store_data(
             'RL',
             dict(
-                sensor=observations[0],
-                action=actions[0],
-                reward=[r / 100.0 for r in rewards]))
+                sensor=observations[0], action=actions[0], reward=rewards))
 
     def _cts_predict(self, observations, states):
         ## each action is already 2D
         assert len(observations) == 1
         actions, _ = self.predict('RL', inputs=dict(sensor=observations))
-        return [actions.values()[0][0]], []
+        a = actions.values()[0][0]
```

Comment:
Do not put this in the agent's code. You should really change the algorithm code.

Reply:
One of the noise used in DDPG is generated by a stochastic process, e.g. `OUNoise`. It is better for each agent to keep its own stochastic process.

Reply:
I don't think the agent should "own" the stochastic process. The reason why we want to have algorithm and model is to separate the computation from the general agent control flow.